template.html

```
@@ -0,0 +1,223 @@
+          <span id="alert-box-button">New Alert</span>
+          <span id="alert-box-text">A new alert has been generated!</span>
+        </div>
+      </div>
+      <table id="overlap" style="border-spacing: 0; margin-top: 10px">
+        <tr>
+          <td style="width: 100%; height: 41px"></td>
+          <td
+            rowspan="2"
+            style="height: 95px; border: 3px solid #cf2c2c; border-radius: 10px"
+          >
+            <div id="alert-box">
```

HTMLHint (1.5.0)

[error] The id value [ alert-box ] must be unique.

(id-unique)